### DISCUSSION TITLE
RustDesk Text is Blurry

### DISCUSSION THREAD

**Discussion by BrooklynSpeed** [rustdesk/rustdesk#7065] [General]
I've been having this strange issue in the past few weeks where in the main window of RustDesk (before I connect to any remote computer) all the text becomes blurry and only clears up when I hover over it with my mouse.

Has anyone experienced this before?

**Comment by jcHuamei** [comment]
me too, Attached photos:
![rustdesk-bug](https://github.com/user-attachments/assets/df8d06cf-610b-4937-afc7-7acc064c800a)

**Comment by southerndoc** [comment]
I've noticed mine is blurry with smaller text/images.  I turned off texture rendering and adaptive bitrate, which has helped.  I'm finding RealVNC has better resolution than RustDesk.  Not sure what they use for compression.  RustDesk is better than RemotePC for image quality though.

UPDATE:  Just compared them side by side.  Using AV1, custom image quality (100%/25fps), hardware acceleration enabled, texture rendering off, and adaptive bitrate off -- RustDesk is more clear than RealVNC.  Performance is almost the same.
